### ISSUE TITLE
pyreverse do NOT sort attribute and method names

### ISSUE THREAD

**Issue by yantaozhao** [pylint-dev/pylint#8776]
### Current problem

pyreverse output uml with class member names sorted.

### Desired solution

do not sort class member names

**Comment by DudeNr33** [comment]
I feel like this is a very subjective thing if sorting is desired or not. So I would like to vote for this:

🚀 keep the sorting
🎉  don't sort, use the order of definition from source code

**Comment by Pierre-Sassoulas** [comment]
Maybe another possibility would be to add an option to be able to do both ?

**Comment by yantaozhao** [comment]
> Maybe another possibility would be to add an option to be able to do both ?

Agree. It would be better to add an option to let the user decide.

**Comment by DudeNr33** [comment]
For additional context: Completely removing the sorting would also be problematic for the existing unit tests, as they require deterministic output.

**Comment by nickdrozd** [comment]
Testing requires deterministic output, but not alphabetical sorting. Sorting by code order would be just as testable. Obtaining that order will be a little trickier, depending on the existing sources of nondeterminism (sets, dictionaries, etc).

Sorting by code order will require some work on somebody's part (somebody other than me, because I don't care either way :sunglasses:).
